site/docs/components/card/examples.mdx

```diff
@@ -50,13 +55,23 @@ You can incorporate actions like [links](../link) or [buttons](../button) into a
 </LivePreview>
 
 <LivePreview componentName="card" exampleName="FullWidthImage" >
+
 ## Full width image
 
 When using images in cards, set the image to span the card's entire width. You can do this by setting `--saltCard-padding: 0` and putting padding around any content below the image.
 
 </LivePreview>
 
+<LivePreview componentName="card" exampleName="BackgroundImage" >
+
+## Image as background
+
+You can set an image as the background of a card using [`background-image`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) in CSS. Make sure there is enough color contrast between the content and the image.
```

Comment:
Can we change "make sure there is enough color contrast between the content and the image" to "Ensure there's enough contrast between the color of the text and the colors of the background image. WCAG recommends a contrast ratio of at least 4.5:1 for regular text and 3:1 for larger text."

Reply:
updated